curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c218c028c56a411b2a379a.md

```diff
@@ -11,17 +11,24 @@ To see your new HTML content for the `targetInputContainer`, you will need to us
 
 The `innerHTML` property sets or returns the HTML content inside an element. Here is an example of how to use it:
```

Comment:
```suggestion
The `innerHTML` property sets or returns the HTML content inside an element. 
```

Reply:
Am I expected to click on "Commit suggestion" here? The changes you suggested look good to me.

Reply:
You can either click on the commit suggestion button and it will automatically apply those changes to your PR.

Or you can make changes locally and push up to the same branch to the same PR.